CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## Unreleased
 
+## 3.2.0 (2022-01-09)
+
+### Changes
+
+* Upgrade Orchard.
```

Comment:
Do we need these in the change log?  I think we should stick with *visible* changes, e.g. only mention that we upgraded orchard if the user is affected in some noticeable way.